Tests/ApolloTests/Interceptors/MultipartResponseSubscriptionParserTests.swift

```diff
@@ -438,7 +438,7 @@ final class MultipartResponseSubscriptionParserTests: XCTestCase {
     let network = buildNetworkTransport(responseData: """
       
       --graphql
-      content-type: application/json
```

Comment:
Shouldn't we keep a test that it works when you just have `application/json` and not the additional directive?

Reply:
We do have those. They're further down in other tests within this file and `MultipartResponseDeferParserTests.swift`. They aren't specifically labelled as content type tests though; that felt superfluous. If you want I can add them though.